queries.sql

```diff
@@ -16,5 +16,92 @@ SELECT * FROM animals WHERE name <> 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+-- Start a transaction for updating
```

Comment:
> Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
- [ ] You do not have the queries for the above requirements in your `queries.sql` file. I'll suggest you add the queries to the file. :smile: :small_airplane: